hw/ip/prim/dv/prim_lfsr/data/prim_lfsr_cov_excl.el

```diff
@@ -0,0 +1,49 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: udij
+// Format Version: 2
+// Date: Mon Oct  5 14:44:08 2020
+// ExclMode: default
+//==================================================
+// Coverage exclusion file for prim_lfsr
+CHECKSUM: "1437046094"
+INSTANCE: prim_lfsr_tb.gen_duts[24].i_prim_lfsr
+ANNOTATION: "[UNR]"
+Assert gen_ext_seed_sva.ExtDefaultSeedInputCheck_A "assertion"
+ANNOTATION: "[UNR]"
+Assert gen_lockup_mechanism_sva.LfsrLockupCheck_A "assertion"
+CHECKSUM: "1437046094 1520894529"
+INSTANCE: prim_lfsr_tb.gen_duts[24].i_prim_lfsr
+ANNOTATION: "[EXTERNAL] This point is fully covered in the FPV bench for prim_lfsr, so it is ok to exclude from DV bench."
```

Comment:
nit: you could make these a bit concise: `[EXTERNAL] Covered by FPV tb.`

Reply:
Good point, I'll update it

Reply:
Do add more meaning explanation to this as applicable - example: 
```
[EXTERNAL] seed input tied off in DV because it is covered in the FPV tb.
```

Reply:
yup that's in progress right now actually (I realized I should have clarified on this exclusion right after I force-pushed)